deps/fdasrsf/dp_grid.c

```diff
@@ -0,0 +1,257 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <math.h>
+#include "dp_nbhd.h"
+#include "dp_grid.h"
+
+
+void dp_all_edge_weights(
+    double* Q1, double* T1, int nsamps1,
+    double* Q2, double* T2, int nsamps2,
+    int dim,
+    double* tv1, int* idxv1, int ntv1,
+    double* tv2, int* idxv2, int ntv2,
+    double* W, double lam)
+{
+    int sr, sc;  /* source row and column */
+    int tr, tc;  /* target row and column */
+    int l1, l2, l3;  /* for multidimensional array mapping */
+    int i;
+
+    for (i=0; i<ntv1*ntv2*ntv1*ntv2; W[i++]=1e6);
```

Comment:
We should put magic numbers in `#define`'s. Also, spaces around operators.

Reply:
Constant INFTY and spaces added in 36dbe63.